analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -25,5 +25,7 @@ public sealed class ObjectsShouldNotBeDisposedMoreThanOnce : ObjectsShouldNotBeD
     public static readonly DiagnosticDescriptor S3966 = DescriptorFactory.Create(DiagnosticId, MessageFormat);
     protected override DiagnosticDescriptor Rule => S3966;
 
-    public override bool ShouldExecute() => false;
+    protected override ILanguageFacade Language => CSharpFacade.Instance;
```

Comment:
SE-based rule should not have access to language facade. It's too powerful tool that should not be available there.

Let's just remove this and solve it during the VB PR